saleor/graphql/payment/tests/benchmark/test_payment_transactions.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from ....tests.utils import get_graphql_content
+
+PAYMENT_TRANSACTIONS_QUERY = """
+query {
+  orders(first:100) {
+    edges {
+      node {
+        payments {
+          id
+          transactions {
+            id
+            isSuccess
+            token
+          }
+        }
+      }
+    }
+  }
+}
+"""
+
+
+@pytest.mark.django_db
+@pytest.mark.count_queries(autouse=False)
+def test_payment_transactions(
+    staff_api_client, orders_for_benchmarks, permission_manage_orders, count_queries
+):
+    content = get_graphql_content(
+        staff_api_client.post_graphql(
+            PAYMENT_TRANSACTIONS_QUERY,
+            permissions=[permission_manage_orders],
+            check_no_permissions=False,
+        )
+    )
+    assert len(content["data"]["orders"]["edges"]) == 11
```

Comment:
The 11 is a kind of magic number, I will suggest comparing this to the count of a given order transaction.